misc/fluent-logger/fluent-logger.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
you should remove the binary file "misc/fluent-logger/fluent-logger" from the pr (can't add comment on the binary so commented here)

Reply:
missing license header for this file